.github/workflows/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+
+ARG NUGET_API_KEY
+ARG PUBLISH_ARGS
+ARG NUGET_PKG
+
+
+COPY ./Source/Zencoder/Zencoder.csproj /build//src/Zencoder/Zencoder.csproj
```

Comment:
Do we need the double // infront of src?

Reply:
Probably not, good spot! 
I'll remove.
Although, this file is copied directly from the existing example provided [here](https://github.com/hudl/dotnet-observability/tree/main/.github/workflows)